### PR TITLE
[add]add hackathon 2022

### PR DESCRIPTION
**Pull request by Candicepan**

### CODE REVIEW

File: events/hackathon2022/guide.md [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+   - Submit your project to the team leader's [GitHub gist](https://gist.github.com/). The leader needs to copy the gist link and click this link to submit your RFC.
+
+1. **Prepare for the contest**
+   - Read our reference materials to get a better understanding of TiDB. 
+   - Have team meetings to improve communication among team members.
+   - On October 19, check the announcement to see if your team is selected for the final.
+   - If you're selected, prepare for the on-site final.
+
+1. **Oct 22 - 23, 2022**     **Code on site** 
+   - Code on site to complete your project. **Your project's source code must be open source and comply with the** **[Apache 2.0 License](https://github.com/pingcap/community/blob/master/LICENSE)****.**
```
Comment (winkyao) [inline events/hackathon2022/guide.md:28]:
```suggestion
   - Code on site to complete your project. **Your project's source code must be open source and comply with the** [Apache 2.0 License](https://github.com/pingcap/community/blob/master/LICENSE)**.**
```

File: events/hackathon2022/guide.md [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+| [Working with TiDB from Java （TiDB v6.x）](https://eng.edu.pingcap.com/catalog/info/id:212) | This course shows you how to implement read and write operations via Java applications. It focuses on the JDBC driver perspective to get you ready for any Java-based ORM frameworks. <br/>1 **Audience:** SQL and Java developers  <br/>2 **Level:** Intermediate  <br/>3 **Duration:** 3 hours **Prerequisites:** Experience with Java programming and SQL  <br/>4 **TiDB version:** v6.x |
+| [Working with TiDB from Python (TiDB v6.x)](https://eng.edu.pingcap.com/catalog/info/id:219) | This course teaches you to implement read and write operations via Python applications. It focuses on the connectors perspective so you'll be ready for any Python-based ORM frameworks. <br/>1 **Audience:** SQL and Python developers  <br/>2 **Level:** Intermediate  <br/>3 **Duration:** 2 hours <br/>4 **Prerequisites:** Experience with Python programming and SQL  <br/>5 **TiDB version:** v6.x |
+
+## RFC template and submission
+
+- RFC template: Click [here](https://github.com/Candicepan/tidbcommunity/tree/master/events/hackathon2022/rfc-template) to check the RFC template.
```
Comment (winkyao) [inline events/hackathon2022/guide.md:58]: check the template link, I got 404

Comment (Candicepan) [reply events/hackathon2022/guide.md]: ok fixed

File: events/hackathon2022/guide.md [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+| [Working with TiDB from Java（TiDB v6.x）](https://eng.edu.pingcap.com/catalog/info/id:212) | This course shows you how to implement read and write operations via Java applications. It focuses on the JDBC driver perspective to get you ready for any Java-based ORM frameworks. <br/>- **Audience:** SQL and Java developers  <br/>- **Level:** Intermediate  <br/>- **Duration:** 3 hours <br/>- **Prerequisites:** Experience with Java programming and SQL  <br/>- **TiDB version:** v6.x |
+| [Working with TiDB from Python (TiDB v6.x)](https://eng.edu.pingcap.com/catalog/info/id:219) | This course teaches you to implement read and write operations via Python applications. It focuses on the connectors perspective so you'll be ready for any Python-based ORM frameworks. <br/>- **Audience:** SQL and Python developers  <br/>- **Level:** Intermediate  <br/>- **Duration:** 2 hours <br/>- **Prerequisites:** Experience with Python programming and SQL  <br/>- **TiDB version:** v6.x |
+
+## RFC template and submission
+
+- RFC template: Click [here](/rfc-template) to check the RFC template.
```
Comment (winkyao) [inline events/hackathon2022/guide.md:58]:
```suggestion
- RFC template: Click [here](rfc-template) to check the RFC template.
```

**Review by winkyao — APPROVED** [review]
LGTM

**Pull request merged**